torch/_tensor.py

```diff
@@ -78,6 +78,22 @@ def _rebuild_from_type_v2(func, new_type, args, state):
 # torch/_C/__init__.pyi.in to add a type annotation for your method;
 # otherwise, it will not show up in autocomplete.
 class Tensor(torch._C.TensorBase):
+    def _clear_non_serializable_data(self):
```

Comment:
Please add a docstring.
In particular mentioning that a subclass that overrides this  MUST call the super() variant